### ISSUE TITLE
Screen reader not detected on macOS

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#12707]
Testing -167159

Version: 1.74.0-insider
Commit: 8b9891739b703b50547f506a7e2bc9565e52beff
Date: 2022-11-29T07:07:14.672Z
Electron: 19.1.6
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: Yes

Automatic screen reader detection doesn't work for me at all, i.e. in none of the scenarios listed in the test plan item. The only way to get in and out of screen reader mode is to use the command.
